.github/workflows/cd.yml

```diff
@@ -48,3 +48,10 @@ jobs:
         with:
           user: __token__
           password: ${{ env.B2_PYPI_PASSWORD }}
+
+  debian-release:
+    name: Build Debian Package
```

Comment:
I think this step should be skipped if `B2_DEBIAN` vars are missing - we won't be running this on our fork and will be running on the upstream